src/lib/arch/riscv/InstructionMetadata.hh

```diff
@@ -2,7 +2,8 @@
 
 #include <string>
 
-#include "capstone/capstone.h"
+#include "simeng/arch/riscv/Architecture.hh"
+#include "simeng/arch/riscv/Instruction.hh"
```

Comment:
Why are these headers now required?

Reply:
Supplies access to `arch::riscv::groupInheritance` and `arch::riscv::Opcode::RISCV_INSTRUCTION_LIST_END` in `ModelConfig.cc` (approach chosen for consistency with already integrated aarch64 work)

Reply:
I don't think this PR should address this, and I am happy with the reason provided above - but including headers for small items such as these creates messy dependancies in the project, something which (I think) should be rectified and cleaned up at some-point

Reply:
The entire include logic based around `InstructionMetadata` needs to be redone. It's an artefact of the old SimEng setups that has been hidden away. Only realised how bad it got during this rebase onto dev